src/Migration/Resource.php

```diff
@@ -52,7 +52,7 @@ abstract class Resource
 
     public const TYPE_HASH = 'hash';
 
-    public const TYPE_ENVVAR = 'envvar';
+    public const TYPE_ENVVAR = 'environment variable';
```

Comment:
As eldad mentioned, in another PR we need to refactor this constant and give it the full type name `TYPE_ENVIRONMENT_VARIABLE`

Reply:
In another PR? I'll create a task